doc/whats_new.rst

```diff
@@ -395,6 +395,8 @@ People who contributed to this release  (in alphabetical order):
 * Teon Brooks
 * Yousra Bekhti
 
+    - Add :func:`mne.beamformer.make_dics` and :func:`mne.beamformer.apply_dics` as well as :func:`mne.beamformer.apply_dics_epochs` to enable the separate computation and application of DICS beamformer weights by `Britta Westner`_, `Alex Gramfort`_, and `Denis Engemann`_.
```

Comment:
this is in the wrong place, also @wmvanvliet is missing

Reply:
... and "Susanna Aro" is referenced in source files

Reply:
Yeah, the what's new needs work. But I'll do that at the very end, since it will continuously cause conflicts with master.